src/create-command-handler.js

```diff
@@ -23,7 +24,12 @@ const createCommandHandler = compiledApp => {
     if (!handler) {
       throw new Error(`Unexpected command ${command}`);
     }
-    return handler(compiledApp, input);
+
+    try {
+      return handler(compiledApp, input);
+    } catch (err) {
+      return handleError(err);
+    }
```

Comment:
This seemed like the best place to catch errors that resulted from calling a `method` like `operation.perform`. Alternatively, we could catch errors in [`middleware`](https://github.com/zapier/zapier-platform-core/blob/master/src/middleware.js#L90) with a `try/catch` as well. 

It'd be nice to have `method` return a `Promise` and then I think we _might_ be able to catch without the `try/catch`.

Put a box around digging into that part though. For now, I think the best place is either here or `middleware`.